### DISCUSSION TITLE
[Inno Setup] 包裝安裝檔時顯示中文 | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#52] [Announcements]
# [Inno Setup] 包裝安裝檔時顯示中文 | 全端開發人員天梯

Inno Setup 內建多種語言的安裝檔，不過內建不支援繁體中文，雖然文件上有非官方語言包可以下載，但還是有一些地雷在，這篇文章就簡單記錄一下。

https://fullstackladder.dev/blog/2022/02/03/inno-setup-utf8-chinese-support/

**Comment by 5idereal** [comment]
嗨！我是 Inno Setup 的譯者，關於編碼問題是我的疏忽，我已經將帶 BOM 的更新檔傳送給作者，應該幾天後就會更新到網頁上。此外，我也有向作者申請內建中文，這樣以後就不用特地上官網下載翻譯檔了。

感謝使用！

**Reply by wellwind** [reply]
太棒了！以後包裝就更方便了！感謝您無私的付出 🥰
